code/controllers/subsystem/spatial_gridmap.dm

```diff
@@ -317,10 +328,14 @@ SUBSYSTEM_DEF(spatial_grid)
 	if(!target_turf)
 		return
 
-	return grids_by_z_level[target_turf.z][ROUND_UP(target_turf.y / SPATIAL_GRID_CELLSIZE)][ROUND_UP(target_turf.x / SPATIAL_GRID_CELLSIZE)]
+	return grids_by_z_level[target_turf.z][GET_SPATIAL_INDEX(target_turf.y)][GET_SPATIAL_INDEX(target_turf.x)]
 
 ///get all grid cells intersecting the bounding box around center with sides of length 2 * range
 /datum/controller/subsystem/spatial_grid/proc/get_cells_in_range(atom/center, range)
+	return get_cells_in_bound(center, range, range)
```

Comment:
muh overhead

Reply:
ahhhhhhhhhhhhhhh my never used debug call is 0.000001% more expensive ahhhhhhhhhhh